fpga_interchange/device_resources.py

```diff
@@ -558,7 +558,7 @@ def __init__(self, device_resource_capnp):
             tile_name = self.strs[tile.name]
             tile_name_index = self.string_index[tile_name]
             assert tile_name not in self.tile_name_to_tile
-            tile_type_index = self.tile_type_to_idx[tile.type]
+            tile_type_index = tile.type
```

Comment:
This seems unrelated to the move to GH actions.

Reply:
Yes, but it is required because the CI would be red.  This is a result of a change in the upstream schema.

Reply:
For clarity, this is in a seperate commit from the other stuff: https://github.com/SymbiFlow/python-fpga-interchange/pull/5/commits/ae1ae41db9082a62bc9750148e04d71802904faa

Reply:
Why the change? What does it fix?

Reply:
Did you read the commit comment?

Reply:
Alright, I'm putting this here in case anyone else looks at this PR:

> Tile type index is now encoded instead of tile type name.

Please update the PR description. These PRs often serve as documentation when I'm trying to understand a change (in addition to Git commit messages.)